src/Chartjs.vue

```diff
@@ -49,7 +45,7 @@ export default {
       this.$nextTick(() => {
         // this.chart.data.datasets = val.datasets
         // this.chart.data.labels = val.labels
-        this.chart.update()
+        this.mountChart()
       })
```

Comment:
I think we should keep the `chart` instance, don't create new instance every times. 
Or add a `force` options for that.